### PR TITLE
Several dependency updates.

### PR DESCRIPTION
**Pull request by alan-agius4**

### CODE REVIEW

File: goldens/size-tracking/aio-payloads.json [hunk 1]
```diff
@@ -2,21 +2,21 @@
   "aio": {
```
Comment (alan-agius4) [inline goldens/size-tracking/aio-payloads.json:2]: The size changes here are caused by the `@angular/material` update.

**Review by AndrewKushnir — APPROVED** [review]

**Review by jessicajaniuk — APPROVED** [review]
reviewed-for: integration-tests, docs-infra, size-tracking

**Review by AndrewKushnir — APPROVED** [review]
Reviewed-for: size-tracking

File: goldens/size-tracking/aio-payloads.json (hunk 1 above)
Comment (devversion) [reply goldens/size-tracking/aio-payloads.json:2]: I believe we never got around reducing the size back to the legacy equivalents. We measured heavily initially and it was significantly worse with the MDC JS code, but now we are only using the CSS so have a little more control now

**Review by devversion — APPROVED** [review]

**Comment by atscott** [comment]
This PR was merged into the repository by commit 208f52209bfc2425c90752e795704c08435d4d77.

**Pull request closed without merging**